### PR TITLE
Add feature language setting.

### PR DESCRIPTION
**Pull request by wdipax**
This pull-request:

- Upgrades gherkingen to v4.0.0 
- Adds golang-gherkingen.feature.language configuration setting.

image

### CODE REVIEW

**Comment by wdipax** [comment]
@hedhyw can you please update the [CHANGELOG.md](https://github.com/hedhyw/vscode-golang-gherkingen/blob/main/CHANGELOG.md) after review?

File: package.json [hunk 1]
```diff
@@ -74,7 +74,185 @@
           "when": "resourceLangId == 'feature'"
         }
       ]
-    }
+    },
+    "configuration": [
```
Comment (hedhyw) [inline package.json:78]: suggestion: what about having version in the configuration?

**Review by hedhyw — APPROVED** [review]
thank you!

File: package.json (hunk 1 above)
Comment (wdipax) [reply package.json:78]: I don't know, what about it?

Comment (hedhyw) [reply package.json:78]: I mean, it's a good idea, created an issue!

https://github.com/hedhyw/vscode-golang-gherkingen/issues/8

**Comment by hedhyw** [comment]
Published

image

image

**Pull request merged**